visualization/app/codeCharta/ui/metricChooser/metricChooser.area.component.html

```diff
@@ -10,15 +10,18 @@
 		md-container-class="ribbonBarDropdown"
 	>
 		<md-select-header>
-			<input
-				id="area-metric-selector"
-				class="metric-search area-metric"
-				ng-model="$ctrl._viewModel.searchTerm"
-				type="text"
-				placeholder="&#xF002; Metric (highest value)"
-				ng-keydown="$event.stopPropagation()"
-				ng-change="$ctrl.filterMetricData()"
-			/>
+			<div class="search-bar">
+				<i class="fa fa-search search-icon"></i>
+				<input
+					id="color-metric-selector"
+					class="metric-search color-metric"
```

Comment:
The id and class color-metric(-selector) has been copy&pasted to every metric chooser.